go.mod

```diff
@@ -229,5 +243,5 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-// See https://github.com/prysmaticlabs/grpc-gateway/issues/2
-replace github.com/grpc-ecosystem/grpc-gateway/v2 => github.com/prysmaticlabs/grpc-gateway/v2 v2.3.1-0.20230315201114-09284ba20446
```

Comment:
I have removed this replace directive for the same reason as https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/38#discussion_r2050349142